figures/5.figure5/README.md

```diff
@@ -0,0 +1,14 @@
+# Generating publication ready figures from the data
+
+## Before you start
+Before running the scripts, to generate figure5, we ran imageJ macros to adjust the lookup tables of each image.
+
+Those macros can be found under the folder `figures/5.figure5/imageJ_macros/`.
+
+Please run the macros before running the scripts.
+
+Please note that we used version 1.54f of imageJ.
```

Comment:
Is there a way to include this specific install for reproducibility or no?

Also, just a recommendation to include link to install any software within READMEs in case anyone may not be as familiar. You also use both ImageJ and FIJI within this README so it might be good to make it clear these are the same.

Reply:
FIJI (Fiji is just imagej)

Reply:
I know what Fiji stands for myself lol I was just recommending in case someone only knows Fiji as Fiji or isn't familiar with the software. @MikeLippincott